build/sync/plan.yml

```diff
@@ -0,0 +1,49 @@
+checks:
+  - type: repo_is_clean
+    params:
+      repo: template
+  - type: repo_is_clean
+    params:
+      repo: plugin
+actions:
+  - paths:
+      - build
+    actions:
+      - type: overwrite_directory
+        params:
+          create: true
+  - paths:
+      - Makefile
+      - .editorconfig
+    actions:
+      - type: overwrite_file
+        params:
+          create: true
+        conditions:
+          - type: file_unaltered
+            params:
+              repo: plugin
+              reference-repo: template
+  - paths:
+      - go.mod
```

Comment:
Is this ready worth hassling with? `go` will do the "right thing" anyway most time and add any missing dependencies anyway.